packages/koa-metrics/README.md

```diff
@@ -0,0 +1,74 @@
+# `@shopify/koa-metrics`
+
+Performance metrics implemented with DogStatsD (<https://docs.datadoghq.com/developers/dogstatsd/>) are used widely across services at Shopify. The standard metrics should be documented and following strong conventions.
+
+This package aims to provide standard middlewares and instrumentation tooling for metrics in Koa.
+
+## Installation
+
+```bash
+$ yarn add @shopify/koa-metrics
+```
+
+## Usage
+
+```javascript
+import Koa from 'koa';
+import metrics from '@shopify/koa-metrics';
+
+const app = new Koa();
+
+app.use(metrics(options));
+```
+
+### Options
+
+The passed in `options` object adheres to the following API:
+
+```javascript
+app.use(
+  metrics({
+    prefix: 'AppName',
+    host: 'some-statsd-host.com:8125',
+  }),
+);
+```
+
+#### `prefix` (required)
+
+The global StatsD metric name prefix; should be provided in `PascalCase`.
+
+#### `host` (required)
+
+The url for the StatsD host; should be provided in the format: `hostname:port`.
+
+## Metrics
+
+We rely on the [hot-shots](https://github.com/brightcove/hot-shots) npm package to send the metrics.
```

Comment:
Do they need to know this?

Reply:
I think so, since the `hot-shots` package provides all the mechanisms for custom timing, etc.